pkg/yurthub/util/connrotation.go

```diff
@@ -79,11 +101,12 @@ func (d *Dialer) Close(address string) {
 	d.mu.Lock()
 	conns := d.addrConns[address]
 	delete(d.addrConns, address)
+	d.addrConns[address] = nil
```

Comment:
Why do you want to add an entry to the map again?

Reply:
fixed